doc/build-unix.md

```diff
@@ -1,30 +1,11 @@
-UNIX BUILD NOTES
+UNIX Build Instructions and Notes
 ====================
-Some notes on how to build Bitcoin Core in Unix.
+Some notes on how to build Elements in Unix.
 
 (for OpenBSD specific instructions, see [build-openbsd.md](build-openbsd.md))
 
-Note
----------------------
-Always use absolute paths to configure and compile bitcoin and the dependencies,
-for example, when specifying the path of the dependency:
-
-	../dist/configure --enable-cxx --disable-shared --with-pic --prefix=$BDB_PREFIX
-
-Here BDB_PREFIX must be an absolute path - it is defined using $(pwd) which ensures
-the usage of the absolute path.
+This will build Elements-Qt as well if the dependencies are met.
```

Comment:
I don't think anyone has built the Qt-enabled binary for some time.  Unless we are sure this works, I don't know that we should include it in the guide.